### PR TITLE
Create src-sdcv.cpp.patch

### PR DESCRIPTION
**Pull request by harieamjari**
Add support for termux prefix.

### CODE REVIEW

File: packages/sdcv/src-sdcv.cpp.patch [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+                 data_dir = stardict_data_dir;
+             else
+-                data_dir = "/usr/share/stardict/dic";
++                data_dir =
++#ifdef __TERMUX__
++                  __TERMUX_PREFIX__
++#else
++                  "/usr"
++#endif
++                  "/share/stardict/dic";
```
Comment (xeffyr) [inline packages/sdcv/src-sdcv.cpp.patch:15]: Everything that you need to do is to change
```
data_dir = "/usr/share/stardict/dic"
```
to
```
data_dir = "@TERMUX_PREFIX@/share/stardict/dic"
```

Please remove unnecessary lines.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/sdcv/src-sdcv.cpp.patch (hunk 1 above)
Comment (harieamjari) [reply packages/sdcv/src-sdcv.cpp.patch]: Done.. That's how it's actually should be done....

**Review by xeffyr — APPROVED** [review]

**Comment by xeffyr** [comment]
When modifying package next time, please bump revision like in https://github.com/termux/termux-packages/commit/fcb1616ed822ad03b656f5d1ee6114d0209f22ba. This is necessary to make package available as update.

**Comment by harieamjari** [comment]
I see... I reviewed https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#rebuilding-package-with-no-version-change and it does states so. I'll keep that in mind in the future.

**Pull request merged**